python_modules/dagster/dagster/_core/events/__init__.py

```diff
@@ -1650,6 +1651,32 @@ def build_asset_materialization_planned_event(
         )
         return event
 
+    @staticmethod
+    def build_asset_check_evaluation_planned_event(
+        job_name: str,
+        step_key: str,
+        asset_check_key: "AssetCheckKey",
+        partition: Optional[str] = None,
+        partitions_subset: Optional["PartitionsSubset"] = None,
+    ) -> "DagsterEvent":
+        """Constructs an asset check evaluation planned event, to be logged by the caller."""
+        event = DagsterEvent(
+            event_type_value=DagsterEventType.ASSET_CHECK_EVALUATION_PLANNED.value,
+            job_name=job_name,
+            message=(
+                f"{job_name} intends to execute asset check {asset_check_key.name} on"
+                f" asset {asset_check_key.asset_key.to_string()}"
+            ),
```

Comment:
do we display these events in the event log? wondering if we even need a message here (we probably do not want to mention __ASSET_JOB__ here either way